src/components/cc-logs-instances/cc-logs-instances.js

```diff
@@ -182,11 +182,17 @@ export class CcLogsInstances extends LitElement {
     }
 
     return html`
-      ${this._renderHeader(i18n('cc-logs-instances.deploying.header'), iconHeaderDeploying)}
-      ${this._renderDeploymentDetails(instances[0].deployment)}
-      <div class="instances">
+      <div class="section section--deploying" role="group" aria-labelledby="deploying-instances">
```

Comment:
**fix:** unless we have a good reason not to, we should use `<fieldset>` + `<legend>` instead of ARIA.

Did you face issues with those?

Reply:
I didn't even try.
I find it much easier to use aria that `<fieldset>` + `<legend>` for which I would need to reset the default CSS.
Also, we recently changed to aria in the `cc-toggle`, so I thought it would be nice to uniformize in other components too.

If you think I'm wrong, I'll change that.

Reply:
I definitely get that and I agree that it's easier to use this technique but since the support on iOS is slightly better with `<fieldset>` + `<legend>`, I think we should stick with that as much as we can.